nativelib/src/main/resources/time.c

```diff
@@ -30,3 +30,28 @@ long long scalanative_nano_time() {
 
   return nano_time;
 }
+
+long long scalanative_current_time_millis() {
+	long long current_time_millis;
+
+#define NANOS_PER_MILLI 1000000
+#define MILLIS_PER_SEC 1000
+#define MICROS_PER_MILLI 1000
+
+#ifdef __MACH__
+	struct timeval  tv;
+	gettimeofday(&tv, NULL);
+	current_time_millis = tv.tv_sec * MILLIS_PER_SEC + tv.tv_usec / MICROS_PER_MILLI;
+#else
+	struct timespec ts;
+    clock_gettime(CLOCK_REALTIME, &ts);
```

Comment:
I used for Mac OS `brew install clang-format` and then `clang-format -i nativelib/src/main/resources/time.c`. There is also a -style=LLVM but I couldn't see any difference in this source.